composer.json

```diff
@@ -34,7 +34,7 @@
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0, <2.2",
```

Comment:
It is related to a change in the constructor of the Symfony\Component\Validator\ConstraintViolation constructor, so all the versions starting with 2.2 are not compatible.